wondrous-app/components/OnboardingDao/styles.tsx

```diff
@@ -0,0 +1,71 @@
+import { InputUnstyled } from '@mui/base';
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const FieldLabel = styled(Typography)`
+  && {
+    color: #ccbbff;
```

Comment:
palette / typography

you can do `import typography from 'theme/typography' ; import palette from 'theme/palette'`